src/main/java/com/inventory/services/OfferService.java

```diff
@@ -39,11 +39,22 @@ public void addOffer(Offer offer) {
         offerRepository.save(offer);
     }
 
-    public void deleteOffer(long id) {
-        offerRepository.deleteById(id);
+    public void deleteOffer(Long id) {
+        int position = -1;
```

Comment:
why? I think this logic is not needed anymore. just deleteById(id) is sufficient for now